nerves_toolchain_ctng/build.sh

```diff
@@ -191,7 +191,7 @@ gcc_tuple()
     # Don't call this until after build_gcc()
     tuplepath=$(ls "$GCC_INSTALL_DIR")
     if [[ -e $tuplepath ]]; then
-        echo "unknown"
+        echo "nerves"
```

Comment:
```suggestion
        echo "unknown"
```

It feels like if somehow that call to `ls` fails that things should start failing.